styles/style_guide/Periods.yml

```
@@ -0,0 +1,7 @@
+extends: existence
+message: "Don't use periods with acronyms or initialisms such as '%s'."
+link: 'https://developers.google.com/style/abbreviations'
+level: error
```

I could see some times where we want to ignore this rule. I think we could make it a warning.

```suggestion
level: warning
```